isolate_example/README.md

```diff
@@ -9,6 +9,24 @@ A sample application that demonstrate best practices when using [`isolates`](htt
 * Demonstrate how to initialize and use an isolate.
 * Show the cost of moving data between isolates and provide alternatives.
 
+## The important bits
+
+### `page_one.dart`
+
+Compares running a large computation on the main isolate with running the same calculation on a second isolate.
+When the main isolate is used, Flutter is unable to render new frames, and the `SmoothAnimationWidget`'s animation freezes.
+### `page_two.dart`
+
+Creates an isolate used to compute an infinate loop of sum of 100M randomly generated numbers while reporting progress. 
+Users can start, terminate, pause, and resume the isolate, as well as modify how the calculation is performed.
+
+### `page_three.dart`
+
+Demonstration how expensive it is to move large amounts of data between isolates and 
```

Comment:
The other two page descriptions begin with present tense verbs ("Compares," "Creates"), so for parallelism's sake this one should be changed to "demonstrates" (or the other two should be changed).